tests/pytorch/nightly/resnet50-mp.libsonnet

```diff
@@ -93,6 +94,67 @@ local utils = import 'templates/utils.libsonnet';
       },
     },
   },
+  local resnet50_tpu_vm = experimental.PyTorchTpuVmMixin {
+    frameworkPrefix: 'pt-nightly',
+    modelName: 'resnet50-mp',
+    paramsOverride: {
+      num_epochs: error 'Must set `num_epochs`',
+      datadir: error 'Must set `datadir`',
+    },
+    command: utils.scriptCommand(
+      |||
+        %(setup_commands)s
+        pip3 install tensorboardX google-cloud-storage
+        python3 xla/test/test_train_mp_imagenet.py \
+          --logdir=$(MODEL_DIR) \
+          --datadir=%(datadir)s \
+          --model=resnet50 \
+          --num_workers=8 \
+          --batch_size=128 \
+          --log_steps=200 \
+          --num_epochs=%(num_epochs)d \
+      ||| % self.paramsOverride,
+    ),
+    podTemplate+:: {
+      spec+: {
+        containerMap+: {
+          train+: {
+            resources+: {
+              requests: {
+                cpu: '1',
```

Comment:
You can use these fields of BaseTest here: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/templates/base.libsonnet#L60

Reply:
On second thought, this probably does not do what you want it to. These cpu/memory requests will apply to the container that runs SSH to connect to the TPU VM, not the TPU VM itself

Reply:
This is carryover from [this change](https://source.cloud.google.com/xl-ml-test/ml-testing-accelerators/+/41f4a3519dbaf53f1ea1f8d5e2bef72e2ab6ca1c) you made to the resnet50 TPUVM test. I assumed it was here to make sure we request a small (Easily available) machine to do the setup commands. Let me know if you want me to remove it

Reply:
I remember now. The base class for this test has a large resource requirement for TPU Nodes, but I lowered it for TPU VM